python/tests/cli/test_validate_command.py

```diff
@@ -23,3 +27,44 @@ def test_validate_command_fails_with_invalid_definition(self):
             assert_validation_failure(result)
             self.assertIn("Failed to validate", result.get_messages_as_string())
             self.assertIn("Undefined type 'invalid'", result.get_messages_as_string())
+
+    def test_validate_command_specify_definition_name_succeeds_with_valid_definition(self):
+        valid_model = create_model_definition("Valid Model", "A valid model.")
+        # TO/DO come back and replace this with test constant definitions
+        valid_schema = create_schema_definition("Test Schema", fields=[create_field_entry("name", "str")])
+
+        TEST_CONTENT = f"{YAML_DOCUMENT_SEPARATOR}{linesep}".join([valid_model.to_yaml(), valid_schema.to_yaml()])
+        with temporary_test_file(TEST_CONTENT) as valid_model_file:
+            result = validate(valid_model_file.name, valid_model.name)
+
+            assert_plugin_success(result)
+            self.assertIn(f"{valid_model_file.name} is valid", result.get_messages_as_string())
+            self.assertIn(valid_model.name, result.get_messages_as_string())
+
+            active_context = get_active_context()
+            self.assertTrue(active_context.is_definition_type(valid_model.name))
+
+            # This check will only work if we allow validate to pollute the active context for validation.
+            self.assertFalse(active_context.is_definition_type(valid_schema.name))
+
+    def test_validate_command_specify_definition_name_fails_with_missing_definition(self):
+        valid_model = create_model_definition("Valid Model", "A valid model.")
+        # TO/DO come back and replace this with test constant definitions
+        valid_schema = create_schema_definition("Test Schema", fields=[create_field_entry("name", "str")])
```

Comment:
This is on hold until I can integrate the new constant test definitions from #452